src/dpp/discordvoiceclient.cpp

```diff
@@ -44,6 +43,8 @@ struct OpusRepacketizer {};
 
 namespace dpp {
 
+constexpr opus_int32 opus_sample_rate_hz = 48000;
```

Comment:
reference to opus_int32 must be wrapped in #HAVE_VOICE
same for any other references to opus or sodium or the lib won't build if voice support is not available

Reply:
Yeah I realized. Changed to `int32_t`.